akka-docs/rst/java/http/routing-dsl/directives/path-directives/path.rst

```diff
@@ -24,6 +24,10 @@ a ``path`` directive will always be empty).
 Depending on the type of its ``PathMatcher`` argument the ``path`` directive extracts zero or more values from the URI.
 If the match fails the request is rejected with an :ref:`empty rejection set <empty rejections>`.
 
+.. note:: The empty string (also called empty word or identity) is a **neutral element** of string concatenation operation,
+ so it will match everything, but remember that ``path`` requires whole remaining path being matched, so (``/``) will succeed
```

Comment:
Ah, sorry, I couldn't see that it was already indented (additionally I thought the requirement was 3 spaces, but 1 may very well be enough)